Automattic/woocommerce-payments
# Refunds API request classes

Fixes #5993
#### Changes proposed in this Pull Request

Refunds API endpoints have been migrated to the request classes. The benefits of the request classes is that we can construct API requests towards server in effortless, providing all the benefits that are already built in with the request classes.

#### Testing instructions

- Make sure that all tests are passing

- Test changed endpoints manually (if you can) by going to the place on the website where are they executed and notice are there any issues with them.
- Make sure to compare previous changes with the new ones and check are there any missed points



-------------------


**Post merge**
